forge-gui/res/cardsfolder/upcoming/star_athlete.txt

```diff
@@ -0,0 +1,14 @@
+Name:Star Athlete
+ManaCost:2 R R
+Types:Creature Human Warrior
+PT:3/2
+K:Menace
+K:Blitz:3 R
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ DBPump | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, choose up to one target nonland permanent. Its controller may sacrifice it. If they don't, Star Athlete deals 5 damage to that player.

```

Comment:
CARDNAME

Reply:
Addressed in [commit](https://github.com/Card-Forge/forge/pull/6225/commits/23d367693b784caffc8cc5cdbf9bd367fb30662c)

Reply:
sorry, I'll try to give you more starting pointers :)
I meant we replace all occurences with CARDNAME (especially in [...]Description$ params) since the engine can then apply name changes from copy effects etc.

Reply:
I appreciate all of the help. I do not mind refining a PR a lot. It is how we get a solid and consistent code base!

Reply:
actually not in the Oracle, since that one is just taken 1:1 for display in "dumber" parts like the deck editor